runtime/src/proposals_configuration/defaults.rs

```diff
@@ -347,3 +347,17 @@ pub(crate) fn unlock_blog_post_proposal() -> ProposalParameters<BlockNumber, Bal
         constitutionality: 1,
     }
 }
+
+// Proposal parameters for the 'Veto Proposal' proposal
+pub(crate) fn veto_proposal_proposal() -> ProposalParameters<BlockNumber, Balance> {
+    ProposalParameters {
+        voting_period: 72200, // The longest grace period
+        grace_period: 0,
+        approval_quorum_percentage: 75,
+        approval_threshold_percentage: 80,
+        slashing_quorum_percentage: 60,
+        slashing_threshold_percentage: 66,
+        required_stake: Some(1_000_000),
+        constitutionality: 0,
```

Comment:
Why the constitutionality is set to zero here? Don't we need at least one council approval?